pkgs/games/blackshades/default.nix

```diff
@@ -1,29 +1,25 @@
-{ lib, stdenv, fetchFromSourcehut
-, SDL, stb, libGLU, libGL, openal, libvorbis, freealut }:
+{ lib, stdenv, fetchgit, zig, glfw, libGLU, libGL, openal, libsndfile }:
 
 stdenv.mkDerivation rec {
   pname = "blackshades";
-  version = "1.3.1";
+  version = "2.4.0";
 
-  src = fetchFromSourcehut {
-    owner = "~cnx";
-    repo = pname;
+  src = fetchgit {
```

Comment:
Please use fetchFromSourcehut

Reply:
Should I try to add fetchgit fallback like fetchFromGithub?  I need to fetch the submodules here.

Reply:
It does not have it already? Please give it a try and if it is more than a 3 line change we can do that later/in a separate PR.

Reply:
Done!